docs/config.md

```diff
@@ -28,15 +28,18 @@ The following fields are supported:
     * Typically named `vmlinuz` or `bzImage`
     * If a relative path is provided, it will be interpreted as relative to
       `vmtest.toml`
+    * Should have `CONFIG_9P_FS` and `CONFIG_NET_9P_VIRTIO` enabled if `rootfs`
+      is in use.
 * `kernel_args` (string)
     * Optional field
     * `kernel` must be specified
     * Additional kernel command line arguments to append to `vmtest` generated
       kernel arguments
 * `rootfs` (string)
-    * Default: `/`
+    * Directory, default: `/`
     * `kernel` must be specified
-    * Path to rootfs to test against
+    * Path to rootfs to test against. Will be mounted as
+      [9p](https://docs.kernel.org/filesystems/9p.html) in the guest.
```

Comment:
Same as above. Or maybe be more vague. "FS suitable for virtualization" or something like that?

Reply:
OK yeah sounds good, but maybe a "such as" is good to help people quickly understand what we're talking about here?